cloud/wfm/src/executor/Executor.ts

```diff
@@ -6,7 +6,7 @@ import Process from '../process/Process';
  *
  * Triggers instantiation and execution of a {@link ProcessInstance}
  */
-interface Executor {
+export interface Executor {
```

Comment:
As per typesript guideline default exports should be avoided. Always export stuff normally to allow * import. defaults aren't imported with star interface. See other PRs